src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.cs

```diff
@@ -17,7 +17,9 @@ internal sealed partial class PortableThreadPool
 
         private const short MaxPossibleThreadCount = short.MaxValue;
 
-#if TARGET_64BIT
+#if TARGET_BROWSER
+        private const short DefaultMaxWorkerThreadCount = 16;
```

Comment:
not 32?

Reply:
there is 
- pthread pool (emscripten level)
- managed thread pool (portion of pthread pool )
- threads allocated by user code (will use the rest of the available pthreads, but even that could be not enough)